frontend/src/components/HeaderDropdownList.vue

```diff
@@ -0,0 +1,94 @@
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-key</v-icon>
+          S'identifier
+        </v-list-item-title>
+      </v-list-item>
+      <v-list-item v-if="!loggedUser && $vuetify.breakpoint.xs" :ripple="false" href="/creer-mon-compte">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-account</v-icon>
+          Créer un compte
+        </v-list-item-title>
+      </v-list-item>
+      <v-divider class="mb-2" v-if="loggedUser || $vuetify.breakpoint.xs"></v-divider>
+      <v-list-item v-if="$vuetify.breakpoint.xs && !this.loggedUser" :ripple="false">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-hammer-wrench</v-icon>
+          Devenir testeur
+        </v-list-item-title>
+      </v-list-item>
+      <v-list-item v-if="loggedUser" :ripple="false" :to="{ name: 'CanteenInfo' }">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-publish</v-icon>
+          Publier
+        </v-list-item-title>
+      </v-list-item>
+      <v-list-item :ripple="false" :to="{ name: 'DiagnosticPage' }">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-chart-pie</v-icon>
+          Autodiagnostic
+        </v-list-item-title>
+      </v-list-item>
+      <v-divider class="mb-2"></v-divider>
+      <v-list-item :ripple="false" :to="{ name: 'CanteensHome' }">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-silverware-fork-knife</v-icon>
+          Nos cantines
+        </v-list-item-title>
+      </v-list-item>
+      <v-list-item :ripple="false" :to="{ name: 'BlogsHome' }">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-newspaper-variant-outline</v-icon>
+          Blog
+        </v-list-item-title>
+      </v-list-item>
+      <v-list-item :ripple="false" href="https://ma-cantine-1.gitbook.io/ma-cantine-egalim/" target="_blank">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-file-document-outline</v-icon>
+          Documentation
+        </v-list-item-title>
+      </v-list-item>
+      <v-list-item :ripple="false" :to="{ name: 'GeneratePosterPage' }">
+        <v-list-item-title class="body-2">
+          <v-icon small color="grey darken-2" class="mt-n1 mr-2">mdi-text-box-check</v-icon>
+          Générer mon affiche
+        </v-list-item-title>
+      </v-list-item>
+      <v-divider class="mb-2"></v-divider>
+      <v-list-item :ripple="false" href="mailto:contact@egalim.beta.gouv.fr" target="_blank">
+        <v-list-item-title class="body-2">
```

Comment:
Even better, we could have the data coming from the script and then doing a v-for, like :

```javascript
data() {
  return {
    buttons: [{
      text: "",
      icon: "",
      href: "",
      to: "",
    }, {
      text: "",
      icon: "",
      href: "",
      to: "",
    },
    ...
   ]
  }
}
```

And iterate with the `v-for` afterwards. It would be clearer and would avoid the need for a new component.

Reply:
https://github.com/betagouv/ma-cantine/issues/256 :+1: